src/core/Table/filters/FilterToggle.tsx

```diff
@@ -50,7 +51,7 @@ export const FilterToggle = <T extends Record<string, unknown>>(
           placement='bottom'
           visible={isVisible}
           onClickOutside={close}
-          appendTo={ownerDocument.body}
+          appendTo={ownerDocument?.body}
```

Comment:
I would have just gone with `() => ownerDocument.body` 😄

Reply:
huh? How that would have helped

Reply:
Huh.
![image](https://user-images.githubusercontent.com/9084735/121188623-4ae1ca80-c837-11eb-98f9-d2a1c43f1488.png)

Well ok 🤷‍♀️

Reply:
Yes, it was in FilterToggle... Initially, wanted to fix only that, but...
Decided to just replace all places not to bother about stuff anymore.